plugins/modules/iso_extract.py

```diff
@@ -154,7 +162,10 @@ def main():
 
     # Use 7zip when we have a binary, otherwise try to mount
     if binary:
-        cmd = [binary, 'x', image, '-o%s' % tmp_dir] + extract_files
+        cmd = [binary, 'x', image, '-o%s' % tmp_dir]
+        if password:
+            cmd += "-p%s" % password
```

Comment:
This is not correct, as `cmd` is a `list` but you are adding a `str` to it. The code should be:
```suggestion
        if password:
            cmd += ["-p%s" % password]
```